subscribers/logs/logger/AppService.cs

```diff
@@ -0,0 +1,87 @@
+/* */
+using Amazon;
+using System;
+using System.Threading;
+using System.Threading.Tasks;
+using System.Net.Http;
+using Microsoft.Extensions.Hosting;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Options;
+using Amazon.SQS;
+using Amazon.SQS.Model;
+using Newtonsoft.Json;
+
+namespace Dta.Marketplace.Subscribers.Logger.Worker{
+    public class AppService : IHostedService, IDisposable {
+        private readonly ILogger _logger;
+        private readonly IOptions<AppConfig> _config;
+        private readonly LoggerContext _loggerContext;
+        private readonly IMessageProcessor _messageProcessor;
+        private AmazonSQSClient _sqsClient;
+        private Timer _timer;
+
+        public AppService(ILogger<AppService> logger, IOptions<AppConfig> config, LoggerContext loggerContext, IMessageProcessor messageProcessor) {
+            _logger = logger;
+            _config = config;
+            _messageProcessor = messageProcessor;
+            _loggerContext = loggerContext;
+        }
+
+        public Task StartAsync(CancellationToken cancellationToken) {
+            _logger.LogInformation("Starting daemon: Logger. {Timer}", new {
+                _config.Value.AwsSqsLongPollTimeInSeconds,
+                _config.Value.WorkIntervalInSeconds,
+                _config.Value.AwsSqsRegion,
+                _config.Value.AwsSqsServiceUrl,
+                _config.Value.AwsSqsQueueUrl,
+                sentryEnabled = string.IsNullOrWhiteSpace(_config.Value.SentryDsn) ? false : true
+            });
+
+            var sqsConfig = new AmazonSQSConfig {
+                RegionEndpoint = RegionEndpoint.GetBySystemName(_config.Value.AwsSqsRegion)
+            };
+            if (string.IsNullOrWhiteSpace(_config.Value.AwsSqsServiceUrl) == false) {

```

Comment:
It might be better to use `!` to test for `false` values.  I can see you've used it elsewhere in this PR.
```suggestion
            if (!string.IsNullOrWhiteSpace(_config.Value.AwsSqsServiceUrl)) {
```